compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -642,16 +674,16 @@ def IREELinalgExt_OnlineAttentionOp : IREELinalgExt_PureOp<"online_attention",
   let description = [{
     Traditional scaled dot product attention computes:
 
-    attention(Q, K, V, scale) = softmax(Q @ K.T * scale) @ V
+    attention(Q, K, V, scale, M?) = softmax(Q @ K.T * scale + M) @ V
 
     Online Attention on the other hand, uses an online normalizer instead of
     softmax:
 
-    online_attention(Q, K, V, scale, running_max, running_sum)
-      = online_normalizer(Q @ K.T * scale, running_max, running_sum) @ V
+    online_attention(Q, K, V, scale, M?, running_max, running_sum)
+      = online_normalizer(Q @ K.T * scale + M, running_max, running_sum) @ V
 
     The advantage of this online_normalizer is that it can be tiled along
-    it's reduction dimension, making the online_attention operator:
+    its reduction dimension, making the online_attention operator:
```

Comment:
same as above, please instead add a line that if mask is specified, the output of the first matmul is modified. sdpa has a particular definition and mask is just a modification on top.